src/pyglottolog/languoids/models.py

```diff
@@ -186,7 +186,31 @@ def _format(self, minimal=True):
 
     @classmethod
     def from_name(cls, name):
+        # some additional common names
+        _code_map = {
+            'burma': 'MM', 'cameroun': 'CM', 'czech republic': 'CZ',
+            'democratic republic of the congo': 'CD', 'drc': 'CD',
+            'east timor': 'TL', 'fsm': 'FM', 'laos': 'LA',
+            'macedonia': 'MK', 'north korea': 'KP',
+            'republic of the congo': 'CG', 'south korea': 'KR',
+            'swaziland': 'SZ', 'trindad and tobago': 'TT',
+            'uae': 'AE', 'uk': 'GB', 'usa': 'US',
+        }
+        if name.lower() in _code_map:
+            res = pycountry.countries.get(alpha_2=_code_map[name.lower()])
+            return cls(id=res.alpha_2, name=res.name)
+
         res = pycountry.countries.get(name=name)
+        if res is None:
+            res = pycountry.countries.get(common_name=name)
+        if res is None:
+            try:
+                for r in pycountry.countries.search_fuzzy(name):
```

Comment:
Do we need a fuzzy search here?

Reply:
The fuzzy search is only needed in cases were the name or official name begins with the search string.
`pycountry.countries.get(name='Iran') is None`
`pycountry.countries.get(common_name='Iran') is None`
but the fuzzy search finds it since the stored name is `Iran, Islamic Republic of`.

One could of course add the 'fuzzy_search' matches explicitly to _code_map.

The best way is to provide this list to the author(s) of the module `pycountry`.

Reply:
I could ask for a PR to `pycountry` changing https://github.com/flyingcircusio/pycountry/blob/master/src/pycountry/databases/iso3166-1.json

Reply:
I looked into the pycountry repo and it seems that many PRs and Issues are active but these have not been processed for over half a year